src/module/actor/data-model-character.js

```diff
@@ -93,109 +99,129 @@ export default class OseDataModelCharacter extends foundry.abstract.DataModel {
       retainer: new ObjectField({
         enabled: new BooleanField(),
         loyalty: new NumberField({ integer: true }),
-        wage: new StringField()
-      })
+        wage: new StringField(),
+      }),
     };
   }
 
   // @todo This only needs to be public until
   //       we can ditch sharing out AC/AAC.
+  // eslint-disable-next-line class-methods-use-this
   get usesAscendingAC() {
-    return game.settings.get(game.system.id, 'ascendingAC');
+    return game.settings.get(game.system.id, "ascendingAC");
   }
 
   get meleeMod() {
     const ascendingAcMod = this.usesAscendingAC ? this.thac0.bba || 0 : 0;
-    return (this.scores.str?.mod || 0) + (this.thac0?.mod?.melee || 0) + ascendingAcMod;
+    return (
+      (this.scores.str?.mod || 0) +
+      (this.thac0?.mod?.melee || 0) +
+      ascendingAcMod
+    );
   }
 
   get rangedMod() {
     const ascendingAcMod = this.usesAscendingAC ? this.thac0.bba || 0 : 0;
-    return (this.scores.dex?.mod || 0) + (this.thac0?.mod?.missile || 0) + ascendingAcMod;
+    return (
+      (this.scores.dex?.mod || 0) +
+      (this.thac0?.mod?.missile || 0) +
+      ascendingAcMod
+    );
   }
 
   get isNew() {
     const { str, int, wis, dex, con, cha } = this.scores;
-    return ![str, int, wis, dex, con, cha]
-      .reduce((acc, el) => acc + el.value, 0)
+    return ![str, int, wis, dex, con, cha].reduce(
+      (acc, el) => acc + el.value,
+      0
+    );
   }
 
   get containers() {
     return getItemsOfActorOfType(
       this.parent,
-      'container',
+      "container",
       ({ system: { containerId } }) => !containerId
     );
   }
 
   get treasures() {
     return getItemsOfActorOfType(
       this.parent,
-      'item',
+      "item",
       ({ system: { treasure, containerId } }) => treasure && !containerId
     );
   }
+
   get carriedTreasure() {
-    let total = this.treasures.reduce((acc, { system: { quantity, cost } }) =>
-      acc + (quantity.value * cost),
+    const total = this.treasures.reduce(
+      (acc, { system: { quantity, cost } }) => acc + quantity.value * cost,
       0
     );
-    return Math.round(total * 100) / 100.0;
+    return Math.round(total * 100) / 100;
```

Comment:
we don't want the floating point number here? I imagine that's why 100.0 is there

Reply:
That was something `prettier-eslint` changed. I don't think using a float here matters; I think JS will parse it as a float if the result comes out as a decimal.